client/src/components/professional/style.css

```diff
@@ -26,14 +26,22 @@
 }
 
 .professionalPaperStyle {
-  background-color: rgba(231,224,215,0.75); 
+  background-color: rgba(231,224,215,0.5); 
+  border-radius: 10px;
   position: relative;
   top: 65vh; 
   bottom: 0;
   width: 65%;
   margin: auto;
 }
 
+.professionalVideo {
+  height: 30vh;
+  width: 40vw;
+  frameborder: 0;
```

Comment:
Move this frameborder as a prop to the iframe tag to get rid of the border around the video.